MapboxVision/VisionManager/VisionManager.swift

```diff
@@ -217,28 +216,17 @@ public final class VisionManager: BaseVisionManager {
     }
 
     override func prepareForBackground() {
-        interruptionStartTime = Date()
         guard state.isStarted else { return }
         isStoppedForBackground = true
         pause()
     }
 
     override func prepareForForeground() {
-        stopInterruption()
         guard isStoppedForBackground else { return }
         isStoppedForBackground = false
         resume()
     }
 
-    private func stopInterruption() {
-        guard let interruptionStartTime = interruptionStartTime else { return }
-
-        let elapsedTime = Date().timeIntervalSince(interruptionStartTime)
-        if elapsedTime >= Constants.foregroundInterruptionResetThreshold {
```

Comment:
suppose we should remove `Constants.foregroundInterruptionResetThreshold` as well